generator/cmd/listresources.ts

```diff
@@ -0,0 +1,108 @@
+import path from 'path';
+import fs from 'fs';
+import { promisify } from 'util';
+import * as constants from '../constants';
+import { series } from 'async';
+import { lowerCaseCompare } from '../utils';
+
+const rootSchemaPaths = [
+    'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json',
+    'https://schema.management.azure.com/schemas/common/definitions.json',
+    'https://schema.management.azure.com/schemas/common/autogeneratedResources.json',
+    'https://schema.management.azure.com/schemas/common/manuallyAddedResources.json',
+];
```

Comment:
Since ARM might not be the only ones consuming this command, we should add a note - maybe in the readme- so users can know the output would be only for the ResourceGroup scoped root schemas.